pkg/system/path.go

```diff
@@ -42,7 +46,7 @@ func DefaultPathEnv(os string) string {
 // a			--> a
 // /a			--> \a
 // d:\			--> Fail
-func CheckSystemDriveAndRemoveDriveLetter(path string, driver pathdriver.PathDriver) (string, error) {
+func CheckSystemDriveAndRemoveDriveLetter(path string, driver isAbser) (string, error) {
```

Comment:
This is an exported function, we shouldn't have an argument that is an unexported type.
Maybe something like `PathVerifier`? (I hate naming things).

Reply:
SGTM, pushed.